### PR TITLE
PexSearch: First implementation

### PR DESCRIPTION
**Pull request by stepanbujnak**

### CODE REVIEW

File: client.go [hunk 1]
```diff
@@ -23,6 +23,20 @@ type Client struct {
...
+	return &Client{
+		c: cClient,
+	}, nil
+}
+
+func (x *Client) getCClient() *C.AE_Client {
+	return x.c
+}
+
+func newClient(typ C.AE_ClientType, clientID, clientSecret string) (*C.AE_Client, error) {
```
Comment (horenmar) [inline client.go:39]:
```suggestion
func newClient(type C.AE_ClientType, clientID, clientSecret string) (*C.AE_Client, error) {
```

Is that intentional or typo?

**Review by horenmar — COMMENTED** [review]
Seems fine, but I don't have enough familiarity with these bindings to make the call.

File: client.go (hunk 1 above)
Comment (stepanbujnak) [reply client.go:41]: intentional, type is a keyword in go.

**Review by romanmatiasko — APPROVED** [review]
LGTM

**Pull request merged**